### PR TITLE
#161748169 toggle category navigation  

### PR DESCRIPTION
**Pull request by joeeasy**
#### What does this PR do?
Toggle category navigation bar 

#### Description of tasks to be completed
- add hamburger icon to category nav on a smaller device
- toggle category nav items on a smaller device 
#### Screenshot 
 
screen shot

#### Relevant Pivotal tracker stories
#161748169

### CODE REVIEW

File: src/components/compounds/CategoriesNav/style.scss [hunk 1]
```diff
@@ -9,3 +9,26 @@
...
+    .categories-item  {
+      display: block;
+      width: 100%;
+    }
+    
+  }
+  .category--brand {
+    display: none
+  }
+}
```
Comment (emmaadesile) [inline src/components/compounds/CategoriesNav/style.scss:56]: Add a new line here

**Comment by johngorithm** [comment]
Good Job @joeeasy 
However, while checking out your fix on its review app, I noticed that at some point, the **x** icon displays instead of the **hamburger** icon when the `dropdown` is not **active**. Of course, this can be misleading and I suggest you put your technical skills together and get it fixed. 👍 

screen shot 2018-11-09 at 4 48 13 pm

**Review by johngorithm — CHANGES_REQUESTED** [review]
👍 

Please handle the requested change

**Review by emmaadesile — CHANGES_REQUESTED** [review]
Great job on making the categories responsive. I observed that the hamburger menu was not responding as it should. Kindly fix it.

**Review by emmaadesile — APPROVED** [review]
LGTM

**Review by johngorithm — APPROVED** [review]

**Pull request merged**